phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -1437,6 +1437,21 @@ function handle_message_list_actions(&$address_list, &$error, $remove_u, $remove
 			$error[] = $user->lang['PM_USERS_REMOVED_NO_PERMISSION'];
 		}
 	}
+
+	/**
+	* Event for additional message list actions
+	*
+	* @event core.message_list_actions
+	* @var	array	address_list		The assoc array with the recipient user/group ids
+	* @var	array	error				The array containing error data
+	* @var	object	remove_u			The variable for removing a user
```

Comment:
`remove_u`, `remove_g`, `add_to`, and `add_bcc` are actually booleans (see https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/includes/ucp/ucp_pm_compose.php#L63). Therefore, please change this to say `bool` instead of `object`.

Reply:
Now sure how that got missed, I swore I changed it.